app/models/concerns/visible.rb

```diff
@@ -0,0 +1,20 @@
+module Visible
+  extend ActiveSupport::Concern
```

Comment:
Why is this extend necessary?

Reply:
This extension is necessary because it tells Rails that concern is being created and will provide an included method into which a block can be passed.